src/config/networking.rnc

```diff
@@ -69,7 +69,9 @@ interface =
     wireless? &
     wifi_settings? &
 
-    dhclient_set_down_link?
+    dhclient_set_down_link? &
+
+    element firewall { text }?
```

Comment:
IIUC the contents should be `"yes" | "no"`. I could only find it in ifcfg-lo so I suspect it is obsolete anyway.